test/dynamo/test_repros.py

```diff
@@ -5752,6 +5752,52 @@ def fn(x):
 
         fn(torch.randn(4))
 
+    # https://github.com/pytorch/pytorch/issues/88813
+    def test_return_value_duplication_grad(self) -> None:
+        def fn(val: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+            return val * 2, val * 2
+
+        x = torch.randn(2, requires_grad=True)
+
+        expect = fn(x)
+        self.assertNotEqual(
+            expect[0].untyped_storage().data_ptr(),
+            expect[1].untyped_storage().data_ptr(),
+        )
+
+        actual = torch.compile(fn)(x)
```

Comment:
use `torch.compile(fn, backend='aot_eager')(x) here. We don't need to run inductor just tracing and AOT autograd.

Reply:
Done!